R/coord-transform.r

```diff
@@ -89,20 +89,27 @@ coord_trans <- function(x = "identity", y = "identity", limx = NULL, limy = NULL
     gg_dep("1.0.1", "`ytrans` arguments is deprecated; please use `y` instead.")
     y <- ytrans
   }
+  if (!missing(limx)) {
+    gg_dep("3.1.0", "`limx` arguments is deprecated; please use `xlim` instead.")
+    if (!is.null(xlim)) {
+      stop("Must not provide both `xlim` and `limx`; please use `xlim` only.")
```

Comment:
Can you rephrase this to a full sentence? (also the one below)

Reply:
I fixed them and corrected the grammar in the `gg_dep()` warnings.